tools/edumfa-migrate-linotp.py

```diff
@@ -332,7 +333,8 @@ def insert_chunks(conn, table, values, chunk_size=100000, record_name="records")
             print('Insert {} {} to {} ...'.format(record_name, chunk,
                                                   min(chunk + chunk_size, values_length) - 1))
             try:
-                conn.execute(table.insert(), values[chunk:chunk+chunk_size])
+                sess.execute(table.insert(), values[chunk:chunk+chunk_size])
```

Comment:
execute command needs to be on Session now, if not, nothing will be commited.

Reply:
Prop a change introduced by SQLAlchemy.